src/nidcpower/metadata/functions.py

```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-# This file is generated from NI-DCPower API metadata version 23.0.0d270
+# This file is generated from NI-DCPower API metadata version 23.0.0d285
```

Comment:
In addition to these 4 metadata files, you should also include the latest exported .proto file (from the grpc export instead of the python export).

Reply:
Updated to the d318 export's proto file.